src/isNonNegative.js

```diff
@@ -0,0 +1,30 @@
+import { both, gte, flip } from 'ramda';
+
+import isNumber from './isNumber';
+
+/**
+ * Checks if value is a non-positive `Number` primitive or object. This includes all positive
+ * numbers and zero.
+ *
+ * @func isNonNegative
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.15.0|v1.15.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isPositive|isPositive}
```

Comment:
`@see {@link RA.isPositive|isPositive}, {@link RA.isNonPositive|isNonPositive}`

Also I would add the following into the `isNegative` source:
`@see {@link RA.isNonNegative|isNonNegative}`